samples/k8s-connected-and-managed-cluster/README.md

```diff
@@ -0,0 +1,155 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Connected and Managed Cluster sample
+
+This folder contains samples that show how an ISV can prepare a Kubernetes application that can be installed on Azure Kubernetes Service (AKS) clusters or Azure Arc-enabled Kubernetes Clusters.
+
+Note:
+  - Throughout this sample:
+      - The terms  "connectedClusters" and "connected clusters" denote Azure Arc-enabled Kubernetes clusters.
+      - The terms  "managedClusters" and "managed clusters" denote Azure Kubernetes Service (AKS) clusters.      
+
+## Artifact changes
+
+The following changes are required to enable support for connected and managed clusters.
+
+### Manifest
+
+Add 'supportedClusterTypes' under extensionRegistrationParameters.
+
+Sample below indicates all distributions of connectedClusters type and all ditributions of managedClusters type are supported.
+
+![Alt text](images/Manifest_changes.PNG)
+
+Following is the guidance on the supportedClusterTypes:
+
+| Property      | Description |
+| -----------   | ----------- |
+| supportedClusterTypes       | Contains an object for each top-level cluster-type. Allowed types are – “managedClusters”, “connectedClusters”. "managedClusters" denotes Azure Kubernetes Service (AKS) clusters. "connectedClusters" denotes Azure Arc-enabled Kubernetes clusters. For each of these cluster types, specify distributions and unsupported Kubernetes versions for these distributions. If supportedClusterTypes is not provided, all distributions of ‘managedClusters’ will be supported by default. If supportedClusterTypes is provided, and a given top level cluster type is not provided, then all distributions and Kubernetes versions for that cluster type will be treated as unsupported. |
```

Comment:
There are so much detail in this table, it seems the table format doesn't look that good.

Reply:
Table does not seem to offer much formatting. For now, let's go with this and consider other ways of organizing this.